calculator.js

```diff
@@ -1,5 +1,53 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+
+function number_validation(number) {
+  if (isNaN(number)) {
+    return false;
+  }
+  else if (number == null || number === ' ') {
+    return false;
+  }
+  else {
+    return Number(number)
+  }
 }
+
+function Operation(num1, num2, operation) {
+  this.num1 = num1;
+  this.num2 = num2;
+  this.operation = operation;
+}
+
+const add = new Operation(1,2,['add','+']);
+const sub = new Operation(1,2,['substract','-']);
+const multiply = new Operation(1,2,['multiply','*']);
+const divide = new Operation(1,2,['divide','/']);
```

Comment:
You only use this for testing.  It likely would have been simpler to just use anonymous objects for these like in `exampleAdditionInput`.

```suggestion
const add = {num1: 1, num2: 2, operation: 'add'};
const sub = {num1: 1, num2: 2, operation: 'subtract'};
const multiply = {num1: 1, num2: 2, operation: 'multiply'};
const divide = {num1: 1, num2: 2, operation: 'divide'};
```